custom/icds_reports/utils/aggregation_helpers/distributed/bihar_api_demographics.py

```diff
@@ -57,7 +57,7 @@ def bihar_state_id(self):
 
     def aggregation_query(self):
         month_start_string = month_formatter(self.month)
-        month_end_string = month_formatter(self.month + relativedelta(months=1, seconds=-1))
+        month_end_string = self.month + relativedelta(months=1, seconds=-1)
```

Comment:
Is there a reason to use this instead of the next month string. To make the expression `< next_month` rather than `<= month end`. It feels a little cleaner and it allows us to consistently use dates instead of mixing dates and datetimes

Reply:
No specific reason. Sure I can change it.